presto-docs/src/main/sphinx/admin/tuning.rst

```diff
@@ -10,29 +10,392 @@ Config Properties
++                                          +                                          +                                    +on each machine whereas distributed joins only need to +
++                                          +                                          +                                    +fit in distributed memory across all machines. This    +
++                                          +                                          +                                    +can also be specified on a per-query basis using the   +
++                                          +                                          +                                    +``distributed_join`` session property.                 +
++------------------------------------------+------------------------------------------+------------------------------------+-------------------------------------------------------+
++   ``node-scheduler.network-topology``    +   ``String`` (``legacy`` or ``flat``)    +             ``legacy``             +Sets the network topology to use when scheduling       +
++                                          +                                          +                                    +splits. ``legacy`` will ignore the topology when       +
++                                          +                                          +                                    +scheduling splits. ``flat`` will try to schedule       +
++                                          +                                          +                                    +splits on the same host as the data is located by      +
++                                          +                                          +                                    +reserving 50% of the work queue for local splits.      +
++------------------------------------------+------------------------------------------+------------------------------------+-------------------------------------------------------+
++       ``task.default-concurrency``       +               ``Integer``                +               ``1``                +Default local concurrency for parallel operators. Use  +
++                                          +                                          +                                    +this as default for sessions properties:               +
++                                          +                                          +                                    +``task_hash_build_concurrency`` and                    +
++                                          +                                          +                                    +``task_aggregation_concurrency`` as well as            +
++                                          +                                          +                                    +``task.join-concurrency`` property.                    +
```

Comment:
Then don't mix session and config properties, because it's hard to read as it is now. There is `task_join_concurrency` session property which you can reference as `p3`.